mypy/nodes.py

```diff
@@ -95,6 +95,7 @@ def get_column(self) -> int:
 
 
 implicit_module_attrs = {'__name__': '__builtins__.str',
+                         '__qualname__': '__builtins__.str',
```

Comment:
I don't think this is a right solution since this way mypy will not complain about `__qualname__` at module scope. The latter is defined only at a class scope.